OtripleS.Portal.Web.Tests.Unit/Services/StudentViews/StudentViewServiceTests.Logic.cs

```diff
@@ -4,6 +4,7 @@
 // ---------------------------------------------------------------
 
 using System;
+using System.Collections.Generic;
```

Comment:
Why do we need this here? please run system clean up across the entire solution.